packages/abstract-sdk/src/cw_helpers/cw_messages.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{AbstractSdkError, AbstractSdkResult};
+
+use abstract_core::proxy::ExecuteMsg;
+use cosmwasm_std::{wasm_execute, CosmosMsg, WasmMsg};
+
+#[derive(Debug, PartialEq)]
+pub enum AbstractMessage {
+    ProxyMessage { msgs: Vec<CosmosMsg>, proxy: String },
```

Comment:
I don't think we should store the proxy address with the messages as it's kind of redundant infomation. 

Imo it makes more sense to change the `Executor` implementation to convert an AbstractMessage into a CosmosMsg. 

I.e. remove the execution trait bound on the APIs and instead just return `AbstractMessage(Vec<CosmosMsg>)`. AbstractMessage should not be convertable to CosmosMsg and should not be able to be included in the Response without converting it with the app. 

The way the developer can convert his AbstractMessages would be through the `Execution` trait which instead of exposing an Executor would directly implement the execute trait on the app/adapter. 

example: 

let mut abstract_msgs: AbstarctMsgs::new();

// collect all the transfers
for coin in coins {
    abstract_msgs.append(app.bank(deps).transfer(...));
}

let proxy_msg = app.execute(deps, abstract_msgs);

Reponse::new().add_message(proxy_msg)

Reply:
I think it would be better to be able to do Response::new()
.add_message(bank.transfer())